app/models/user.rb

```diff
@@ -174,7 +174,7 @@ def permitted_correspondence_types
 
   def other_teams_names(current_team)
     self.teams.delete(current_team)
-    self.teams.map(&:name).to_sentence
+    self.teams.order(:id).map(&:name).to_sentence
```

Comment:
not clear why this change was made?

Reply:
Roll back this code as it is not clear why this was added (intent)